### PR TITLE
Feature/GitHub actions

### PR DESCRIPTION
**Pull request by abielawa**
- Added Github Actions workflow with Slack notifications
- Few syntax fixes

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: "Mqperf terragrunt tests"
+
+on:
+  push:
+    branches:
+      - "v2"
+
+env: 
+  TF_VAR_GCP_PROJECT: ola-sandbox
```
Comment (Sztigar) [inline .github/workflows/ci.yaml:9]: I think it's a good practice to use separate account for testing. We should discuss this with @gkocur.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          credentials_json: "${{ secrets.GCP_SA_KEY }}"
+      - name: "Setup Google Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@v0"
+        with:
+          project_id: ${{ env.TF_VAR_GCP_PROJECT }}
+      - name: Setup terraform
+        uses: hashicorp/setup-terraform@v1
+        with: 
+          terraform_wrapper: false
+      - name: Test terraform
```
Comment (Sztigar) [inline .github/workflows/ci.yaml]: Are this `test` steps necessary, or are they leftovers from debugging?

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.7.9'
+      - name: Install python dependencies 
+        run: pip install --upgrade jsonpath-ng
+      - name: Setup Terragrunt
+        uses: autero1/action-terragrunt@v1.1.0
+        with:
+          terragrunt_version: latest
+      - name: Test terragrunt
```
Comment (Sztigar) [inline .github/workflows/ci.yaml]: Same as above.

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        run: terragrunt -version
+      - name: Remove cache
```
Comment (Sztigar) [inline .github/workflows/ci.yaml]: I don't think this step does anything. From what I understand there is no cache in GHA, unless specifically added in previous runs/steps.

File: .github/workflows/ci.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-central-1
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v0
+        with:
+          credentials_json: "${{ secrets.GCP_SA_KEY }}"
+      - name: "Setup Google Cloud SDK"
```
Comment (Sztigar) [inline .github/workflows/ci.yaml]: I'm wondering if we need to setup Google Cloud SDK? Previous step authenticating isn't enough for our case?

**Review by Sztigar — APPROVED** [review]
Added some questions to workflow, other than that LGTM.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (abielawa) [reply .github/workflows/ci.yaml]: Fair point, they are leftovers from debugging - can be removed

File: .github/workflows/ci.yaml (hunk 5 above)
Comment (abielawa) [reply .github/workflows/ci.yaml]: Same as above - it's a leftover from a step where I had to enable container registry api - also can be removed

**Pull request merged**